vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -2,19 +2,19 @@
 sidebarDepth: 2
 ---
 
-# Bundles Upgrade Guideline
+# Bundle Upgrade Guideline
 
-Two types of bundles can be utilized on Entando 7.1+. The original git-based bundle added a composable method with reusable components. The newly introduced docker-based bundles optimize their structure and expands functionality. The [Bundle Evolution Comparison](./bundle-comparison.md) table lists the differences between the two types. The following page describes the guidelines for upgrading to docker-based v5 bundles.
-
-Git-based bundles do not need to be to upgraded. Both types of bundles are compatible on Entando 7.1+, but docker-based bundles provide several innovations and an improved developer experience.
+This guideline describes the process for upgrading a git-based bundle to the newly introduced docker-based bundle (v5). 
```

Comment:
*** Am trying out 'v5' to stand in for docker-based sometimes. Got really sick of saying docker-based everywhere. I think v5 is much easier to use if introduced properly.